src/components/account/myAccount.js

```diff
@@ -1,25 +1,22 @@
 /** @jsx jsx */
-import React from 'react'
+import React, { useState, useEffect } from 'react'
 import { TorusContext } from '../../contextProvider/torusProvider'
-import { ProjectContext } from '../../contextProvider/projectProvider'
-import Pagination from 'react-js-pagination'
-import styled from '@emotion/styled'
-import theme from '../../gatsby-plugin-theme-ui'
-import {
-  Avatar,
-  Badge,
-  Button,
-  Box,
-  Input,
-  Flex,
-  Spinner,
-  Text,
-  jsx
-} from 'theme-ui'
+import { Avatar, Button, Box, Flex, Text, jsx } from 'theme-ui'
 
-export const MyAccount = () => {
-  const { user } = React.useContext(TorusContext)
-  console.log({ user })
+const MyAccount = ({ info }) => {
+  const [ethPrice, setEthPrice] = useState(1)
+  const { balance, user } = React.useContext(TorusContext)
+
+  useEffect(() => {
+    const init = async () => {
+      fetch(
+        'https://min-api.cryptocompare.com/data/price?fsym=ETH&tsyms=USD,EUR,CNY,JPY,GBP'
```

Comment:
earmarking this. currency should probably be selectable by user (as a global setting?)

or do we only support USD?

Reply:
I am using this api to show a quick reference point for the ETH price. I agree we should put a setting for default FIAT currency on the app. We can discuss with more details on what settings we'll provide in an specific issue